src/commands/kv/bucket/mod.rs

```diff
@@ -15,22 +15,24 @@ use std::path::Path;
 
 use cloudflare::endpoints::workerskv::write_bulk::KeyValuePair;
 
-use walkdir::{DirEntry, WalkDir};
+use ignore::overrides::{Override, OverrideBuilder};
+use ignore::{Walk, WalkBuilder};
 
+use crate::settings::target::Target;
 use crate::terminal::message;
 
 // Returns the hashed key and value pair for all files in a directory.
 pub fn directory_keys_values(
+    target: &Target,
     directory: &Path,
     verbose: bool,
 ) -> Result<(Vec<KeyValuePair>, AssetManifest), failure::Error> {
     let mut upload_vec: Vec<KeyValuePair> = Vec::new();
     let mut asset_manifest: AssetManifest = AssetManifest::new();
 
-    for entry in WalkDir::new(directory)
-        .into_iter()
-        .filter_entry(|e| !is_ignored(e))
-    {
+    let dir_walker = get_dir_iterator(target, directory)?;
```

Comment:
nit: is it a dir_walker or a dir_iterator?

Reply:
i guess they are kind of the same thing--the walker iterates through the directory entries :)